app/main.py

```diff
@@ -1,8 +1,53 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
```

Comment:
fix formatting. Brackets should be located in one of two options: open and start with the text or have a line break between the text